Ginger/Ginger/Actions/AddActionPage.xaml.cs

```diff
@@ -110,7 +110,9 @@ private void LoadGridData()
                     {
                         LegacyActions.Add(cA);
                     }
```

Comment:
why we still have the IsLegacyAction flag? we said we will replace it with the LegacyActionPlatformsList but currently we have both. please fully replace IsLegacyAction  with LegacyActionPlatformsList